### PR TITLE
Updates to contributing.md

### PR DESCRIPTION
**Pull request by jfdm**
1. Made explicit rational behind the libraries in `libs`.
2. Made explicit where contributions shoule be made.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,18 +28,28 @@ Please clean up any messes that you find, and don't leave behind new messes for
...
 Likewise, the contents of base are available with no special options.
-The other two packages that ship with Idris, contrib and effects, require a the use of the `-p` command-line argument to bring their contents into the include path.
+The other two packages that ship with Idris, contrib and effects, require the use of the `-p` command-line argument to bring their contents into the include path.
 New contributions should probably be sent to contrib first, so that they can get maintained with the Idris distribution.
 If they turn out to be widely applicable and useful, they may later be moved into base.
 
 As Idris is still being developed we are open to suggestions and changes that make improvements to these default packages.
-Major changes to the library (or Idris itself) should ideally be discussed first through the projects official channels of communication i.e. the mailing list, github wiki, or IRC, or as a [Dragon Egg](https://github.com/idris-lang/Idris-dev/wiki/Feature-proposals).
+Major changes to the library (or Idris itself) should ideally be discussed first through the projects official channels of communication:
```
Comment (david-christiansen) [inline CONTRIBUTING.md]: Shouldn't it say "project's" (with apostrophe)?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,18 +28,28 @@ Please clean up any messes that you find, and don't leave behind new messes for
...
++ `base` is tried and tested code that may be useful, and has seen active use in multiple projects.
++ `contrib` is code that is experimental in design and want to test before possible inclusion in `base`.
++ `effects` is an implementation of algebraic effects that is packaged alongside Idris.
+
 These packages should not be seen as the *standard* as when working with dependent types we do not necessarily know how best to work with dependent types.
 These packages offer functionality that can be built on top of when constructing Idris programs.
 
-One major point to make is that everything in prelude will be imported automatically, unless given
-the `--noprelude` option.
+One major point to make is that everything in prelude will be imported automatically, unless given the `--noprelude` option.
```
Comment (david-christiansen) [inline CONTRIBUTING.md]: Perhaps we should remove the bit "One major point to make is that "? It seems unnecessary to the meaning of the sentence.

**Comment by david-christiansen** [comment]
I just slapped some nitpicky comments on here. What do you think?

**Comment by jfdm** [comment]
I do not disagree. I think what you are seeing are long standing issues. Will update the PR

**Pull request merged**